app/src/main/java/com/github/steroidteam/todolist/ItemViewActivity.java

```diff
@@ -71,29 +106,67 @@ public void addTask(View view) {
         newTaskET.getText().clear();
     }
 
-    private class TasksAdapter extends ArrayAdapter<Task> {
-        public TasksAdapter(Context context, List<Task> users) {
-            super(context, 0, users);
+
+    private class todoAdapter extends BaseAdapter {
+
+        private final TodoList todoList;
+
+        public todoAdapter(TodoList todoList) {
+            this.todoList = todoList;
+        }
+
+        public void add(Task task) {
+            todoList.addTask(task);
```

Comment:
I believe that there's a `this.notifyDataSetChanged()` missing here. This causes the view not to update when a task is created.